app/src/main/java/com/github/swent/echo/compose/map/MapViewProviderImpl.kt

```diff
@@ -78,6 +78,11 @@ class MapLibreMapViewProvider : IMapViewProvider<MapView> {
         }
     }
 
+    override fun setSavedCameraPosition(newPosition: LatLng, zoomLevel: Double) {
+        lastCameraPosition =
+            CameraPosition.Builder().target(newPosition).zoom(zoomLevel).bearing(0.0).build()
```

Comment:
Knitpick: Isn't .bearing(0.0) redundant here? Is the default value of the bearing not 0?

Reply:
You're right, I'll change that :)